src/test/kotlin/org/rust/ide/template/postfix/IfPostfixTemplateTest.kt

```diff
@@ -110,6 +110,8 @@ class IfPostfixTemplateTest : PostfixTemplateTestCase(IfExpressionPostfixTemplat
                 }
             }
             """
+            ,
+            checkSyntaxErrors = false
```

Comment:
This should actually be `true` :) There is a `;` missing after let. The test should be

```
             fn main() {
                 let a = if (true) {
                     true == false.if<caret>
                 } else {
                     false == true
                  }; // <-
              }
```